app/models/daos/JobDAO.scala

```diff
@@ -17,25 +17,28 @@ class JobDAO @Inject()(dbapi: DBApi)(implicit ec: DatabaseExecutionContext) {
 
   implicit private val parameters = Macro.toParameters[Job]
 
+  def findAll(user: User): Future[Seq[Job]] = Future {
+    val idName = user.userType match {
+      case UserType.Candidate => "candidate_id"
+      case UserType.Recruiter => "recruiter_id"
+    }
+
+    db.withConnection { implicit conn =>
+      SQL"""
+        #$selectJobModelFields
+        FROM jobs
+        WHERE #$idName = ${user.userID}
+      """.as(jobRowParser *)
+    }
+  }
+
   def find(id: Int): Future[Option[Job]] = Future {
     db.withConnection { implicit conn =>
-      SQL("""
-        SELECT
-          id,
-          role,
-          company,
-          location,
-          salary,
-          compensation,
-          description,
-          benefits,
-          viewed,
-          submitted_at as submittedAt,
-          candidate_id as candidateId,
-          recruiter_id as recruiterId
+      SQL"""
+        #$selectJobModelFields
```

Comment:
what is this `#` for?

Reply:
Think for literal string interpolation / prepared statement vs. pass as parameter. Was going by this:

https://www.playframework.com/documentation/2.7.x/ScalaAnorm#SQL-queries-using-String-Interpolation